test/es6/module-syntax.js

```diff
@@ -9,21 +9,21 @@ WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
 
 function testModuleScript(source, message, shouldFail) {
     let testfunc = () => WScript.LoadModule(source, 'samethread');
-    
+
     if (shouldFail) {
         let caught = false;
-        
+
         // We can't use assert.throws here because the SyntaxError used to construct the thrown error
         // is from a different context so it won't be strictly equal to our SyntaxError.
         try {
             testfunc();
         } catch(e) {
             caught = true;
-            
+
             // Compare toString output of SyntaxError and other context SyntaxError constructor.
             assert.areEqual(e.constructor.toString(), SyntaxError.toString(), message);
```

Comment:
@boingoing @Yongqu hey guys fyi the arguments are reversed on the assert.areEquals calls in this file